test/ggsn_proxy_SUITE.erl

```diff
@@ -711,6 +712,10 @@ init_per_testcase(request_fast_resend, Config) ->
 			     meck:passthrough([Request, Msg, Resent, State, Data])
 		     end),
     Config;
+init_per_testcase(path_maint, Config) ->
+    set_path_timers(#{'echo' => 700}),
```

Comment:
Is it worth saving the old value in Config? Could be used by end_per_testcase/2 later.

Reply:
Yes, but this whole set_path_timers thing came from an earlier change and I don't want to include too much changes in this simple fix.